apps/aklite-offline/cmds.cpp

```diff
@@ -19,10 +19,15 @@ int CheckCmd::checkSrcDir(const po::variables_map& vm, const boost::filesystem::
 
 int InstallCmd::installUpdate(const po::variables_map& vm, const boost::filesystem::path& src_dir,
                               const std::string& target_name, bool force_downgrade) const {
-  AkliteClientExt client(vm, false, false);
+  AkliteClientExt client(vm, false, false, false);
```

Comment:
Not related to this PR. I think we should replace these boolean params with bitwise flags defined as an enum values, hence only one parameter is passed to the ctor (enum) and a caller defines required params through bitwise AND, e.g. (READ_ONLY | CHECK_POST_CB).